### PR TITLE
Travis JDK 11 Experiment

### PR DESCRIPTION
**Pull request by yschimke**
Curious what breaks when building with JDK 11.  Also the oss parent pom is now deprecated.

### CODE REVIEW

**Comment by swankjesse** [comment]
Something we need to look out for. SNI + TLS 1.3 + session resumption is broken in early versions of JDK 11. 
https://bugs.openjdk.java.net/browse/JDK-8211806

**Comment by swankjesse** [comment]
Request peol

**Comment by swankjesse** [comment]
Request people upgrade. It's fixed in 11.02.

File: pom.xml [hunk 1]
```diff
@@ -290,6 +419,85 @@
...
+              <groupId>org.apache.maven.plugins</groupId>
+              <artifactId>maven-surefire-plugin</artifactId>
+              <configuration>
+                <argLine>-Xbootclasspath/p:${bootclasspathPrefix} -Xms512m -Xmx512m</argLine>
+              </configuration>
+              <dependencies>
+                <dependency>
+                  <groupId>org.mortbay.jetty.alpn</groupId>
+                  <artifactId>alpn-boot</artifactId>
+                  <version>${alpn.jdk7.version}</version>
```
Comment (yschimke) [inline pom.xml]: Bad merge, sorry

**Pull request closed without merging**